### ISSUE TITLE
Blink again when currently on the bracket and hotkey is pressed

### ISSUE THREAD

**Issue by ro31337** [pvorona/atom-bracket-highlight#1]
(it sounds more like feature request).

When you are on the bracket, and the pair bracket is out of screen, user usually scrolls to find where it is.  It's hard to spot it, because the animation is already over at this moment. It would be nice to have a hotkey (like `Ctrl` key) to enable this animation again. So you can scroll, and keep pressing hotkey to find where it is.

**Comment by pvorona** [comment]
@ro31337 Yeah, i know what are you talking about. In this case I'm using bracket-matcher:go-to-matching-bracket to see where its hiding, give it a try.

**Comment by ro31337** [comment]
Yep, but I think it's more useful to have it highlighted IMO. I don't wanna jump to matching bracket, because sometimes I can't go back. For example:

```
        this.fm.blabla((error) => { // <-- Put your cursor here, but before the bracket, and click Ctrl+m two times
          this.errorHandler(error);
          if (!error) {
            // blabla
          };
        });
```

So it's easier to see it just highlighted again. I have zero knowledge about atom plugins, probably will need just for this one and add functionality. Do you think it can be implemented with styles only? I think not, some coffee coding is required.

**Comment by pvorona** [comment]
@ro31337 I've never faced with such a case, so yeah you are right. I probably should add an option to toggle highlighting or think for another way to highlight brackets. For now I see highlighting line numbers with matching brackets is also good idea

**Comment by matrixik** [comment]
Highlight matching line was added in built in bracket-matcher
 https://github.com/atom/bracket-matcher/pull/290
